### DISCUSSION TITLE
Contributing docs from other repositories: Any license requirements?

### DISCUSSION THREAD

**Discussion by theletterf** [open-telemetry/opentelemetry.io#3814] [Other]
While I'm sorting out https://github.com/open-telemetry/opentelemetry.io/pull/3459 with my employer to see how we can license the content properly for upstreaming, I've a more general question concerning docs contributions:

- Does OpenTelemetry.io have any license requirements when it comes to accepting docs from other repositories?
- Would content coming from a CC-licensed repository be used? What about images with copyrighted or patented elements in it?
- Would we need to use https://opentelemetry.io/docs/acknowledgements/ for open source credits? Couldn't a `THIRD-PARTIES` text file do it?

**Comment by svrnm** [comment]
Some history to the acknowledgements page: https://github.com/open-telemetry/opentelemetry.io/issues/1201

I don't have immediate answers to your questions, except that I think we _should_ be able to accept docs from 3rd parties as long as they have a license that allows us to (a) use them and (b) remix them.

**Comment by theletterf** [comment]
Any news on this? Do we have a Legal SIG? :P

**Comment by svrnm** [comment]
CC-by licensed files are something we can accept (and have accepted in the past?), all we need to do is a proper attribution, ideally on the pages where we use them, or at a place like the Acknowledgements, etc. 

> What about images with copyrighted or patented elements in it?

This is much more complicated. In short I would say, that if they can not be re-licensed to a CC-BY or compatible, we can not accept them, even if we get "exclusive" rights to use them, because our docs have an open license and I don't want to get people into trouble by taking the "wrong" parts of our website.

**Comment by theletterf** [comment]
FWIW, we've gotten an OK from our org to publish https://github.com/open-telemetry/opentelemetry.io/pull/3459. Future cases will be reviewed individually, which is OK for us for now.
